pitch.md

```diff
@@ -23,38 +89,146 @@ should be more specific than "The user describes properties of unicorns.", e.g.,
+graphs for other purposes, thought it'd take considerably more effort to do.  It
+should be impossible to do algorithms homework, build websites, and do most other
+things done in general purpose programming languages with this DSL.
 
 ## Related work
 _Are there any other DSLs in this domain? If not, describe how you know there
 aren't and conjecture why not. If so, describe them and provide links. How well 
 do they address the need? Are there any particularly admirable qualities of the
 language? Are there parts of the language you think could be improved?_
 
+There are no other DSLs in this domain that I could find. I was able to find
+some apps that addressed the need for a simple graph maker, such as 
+[Economics Graph Maker](https://www.economicsgraphmaker.com/) and [GraphTool](http://harmon.uconn.edu/GraphTool/). I looked into widgets and extensions for note
+taking apps, specifically Jupyter notebook and Microsoft OneNote, and wasn't able
+to find any DSL.  These seem like the likely places that I'd find a DSL that
+integrates with note-taking, so I am quite sure there aren't any out there.
+I don't think there's been one created because extensible note-taking apps seem
+to be relatively new.  Furthermore, this is such a specific domain, and I think
+most people are focused on creating extensions that can appeal to a wide variety
+of people.  I also think creating apps to solve problems is more common than
+creating a language, which is why we can find apps but not a DSL to solve this
+problem. 
 
 ## Suitability
 _What percentage of your time do you think will be spent directly engaging in
 the **language** aspects of this project (e.g., making language design
 decisions), as opposed to "systems" aspects of the project (e.g., implementing a
 complicated semantics that doesn't require a lot of language design)?_
 
+This is a difficult one to gauge.  Because we're trying to make it as easy as 
+possible for a user to build economics graphs, much of the essential functionality
+could be solved through individual function calls like`SupplyCurve()`.  However, 
+deciding on how to best design customization and transformations on graphs could
+prove a significant challenge.  Balancing ease and speed with users' freedom to
+express the graphs they want could require a considerable  amount of attention 
+and time.
+
+In terms of systems aspects of the project, the DSL needs to run in some 
+note-taking application.  Thus, my concern with systems programming lies in 
+whether these applications' APIs will provide the functionality that I need and
+are easy to work with. If I can find a notebook app that is extensible and 
+provides enough access to functionality such that building in a DSL is trivial,
+I will have very litle time spent on systems aspects of the project.  However,
+I am not sure whether that will be the case, so it really depends on the apps
+that I find.        
```

Comment:
It definitely sounds like you can go to town on the design stuff. I hope you find a good note-taking app so you don't have to worry about the backend!